docs/platform/disk-images/how-to-initiate-a-cross-data-center-migration-for-your-linode/index.md

```diff
@@ -18,7 +21,7 @@ You can migrate your Linode across data centers using the Linode Cloud Manager.
 We recommend using our [Choosing a Data Center](/docs/platform/how-to-choose-a-data-center/) guide to learn how to choose and speed test a data center.
 {{< /note >}}
 
-This guide will cover:
+## In this Guide:
 
 - [Important details to know before initiating a cross data center migration](/docs/platform/migrating-to-a-different-data-center/#things-to-know-before-migrating).
```

Comment:
Can these just be shortened to anchors? Or, update the links to point to the new guide URL slug?

Reply:
Updated